src/pages/login/components/LoginForm.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { FlexBox } from '~/components/Box';
+import { useSelector } from '~/store/hooks';
+import { FormRow } from '~/components/forms/FormRow';
+import { Button } from '~/components/Button';
+import { FormProvider, useForm } from 'react-hook-form';
+
+export interface formData {
```

Comment:
Interfaces and types in general, should be named with UpperCammelCase - otherwise, it's difficult to distinguish them from normal variables. There are also conventions where every interface is prefixed with letter `I` (e.g. IFormData in this case) to distinguish them from class names, although there is lots of discussion about it (https://stackoverflow.com/questions/5816951/prefixing-interfaces-with-i). Moreover, it would be nice to add a little more descriptive name to it, e.g. `LoginFormData`. 
```suggestion
export interface LoginFormData {
```